packages/app-desktop/gui/PdfViewer.tsx

```diff
@@ -64,6 +66,16 @@ export default function PdfViewer(props: Props) {
 		menu.popup(bridge().window());
 	}, [props.dispatch]);
 
+	const saveFile = useCallback(async (file: Uint8Array) => {
+		const tempPath = `${tmpdir()}/${props.resource.id}.pdf`;
+		await shim.fsDriver().writeFile(tempPath, file);
+		console.log('tempPath', tempPath);
```

Comment:
Could you remove all these console statements please - here and across the other changes